deployment/terraform_azure/mongodb.tf

```diff
@@ -0,0 +1,38 @@
+resource "azurerm_cosmosdb_account" "db" {
```

Comment:
`db` is vague especially if there is more than one database. Please use a more descriptive name.